metrics/metrics.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017, Square, Inc.
+
+// Package metrics is a wrapper of github.com/rcrowley/go-metrics.
+package metrics
+
+import (
+	gometrics "github.com/rcrowley/go-metrics"
+)
+
+type Metrics interface {
+	GetOrRegister(string, interface{})
```

Comment:
Is this method overloaded? i.e. doing multiple, discrete things? Also, what's being gotten if there's no return value?

Reply:
I misread the docs, it is supposed to return something. I'll rework it. But it is supposed to do get or register in one call because that makes it threadsafe, so it's not overloaded in that sense.